src/test/scala/mesosphere/marathon/test/MarathonTestHelper.scala

```diff
@@ -460,7 +460,10 @@ object MarathonTestHelper {
 
       def withPortMappings(newPortMappings: Seq[PortMapping]): AppDefinition = {
         val container = app.container.getOrElse(Container.Mesos())
-        val docker = container.docker.getOrElse(Docker(image = "busybox")).copy(portMappings = newPortMappings)
+        val docker = (container match {
+          case c: Docker => c
+          case _ => Docker(image = "busybox")
+        }).copy(portMappings = newPortMappings)
```

Comment:
This is some funky test method. I just expanded `container.docker`. I really do not know what the author intended here.